db/test-data/tests-source-snapshot-insert.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet id="tests-source-snapshot-insert" author="nulls" context="dev">
```

Comment:
Let's load the data from the .csv file. It's shorter and more convenient.

Reply:
migrated `tests-source-snapshot-insert.xml` and `tests-source-version-insert.xml`